### PR TITLE
Add Satismeter survey to email editor [MAILPOET-5911]

### PR DESCRIPTION
**Pull request by lysyjan**
## Description

_N/A_

## Code review notes

_N/A_

## QA notes

1. Create and save at least two emails in the new email editor
2. Open one of the new emails in the new email editor
3. A new Satismeter poll should be visible

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5911]

## After-merge notes

_N/A_

## Tasks

- [ ] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [ ] I added sufficient test coverage
- [ ] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5911]: https://mailpoet.atlassian.net/browse/MAILPOET-5911?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pavel-mailpoet — DISMISSED** [review]

**Comment by veljkho** [comment]
Hi @lysyjan 
I'm trying to test this but I do not see the poll after saving or sending the new email editor, using fresh new site.
Is it only by saving the email editor (saving as draft) or I could also send the newsletter twice time and then the third time creating a new newsletter using the email editor I should see the poll? 

Additionally, what about this statement from the Jira ticket: Display the NPS after leaving the editor to ensure the user has finished creating their first email.

Many thanks.

**Comment by lysyjan** [comment]
@veljkho I forgot to add one condition to the QA instructions. Because Satismeter is a third-party library, you need to allow MailPoet third-party libraries on your site. Can you check it, please?

> Additionally, what about this statement from the Jira ticket: Display the NPS after leaving the editor to ensure the user has finished creating their first email.

I guess for this reason, you need to have at least two new emails. They don't have to be sent, but the first should be created and saved before the poll is displayed.

**Comment by Aschepikov** [comment]
@lysyjan I couldn't trigger this on my site too,  loading of the third-party libraries is enabled. I wanted to check it on a new install, maybe later today

**Comment by lysyjan** [comment]
@Aschepikov I tried the build on my staging site, and it worked as expected.

**Comment by Aschepikov** [comment]
@lysyjan thank you I will retest it

**Comment by lysyjan** [comment]
Thank you @Aschepikov. Ping me on Slack when you need assistance or when it doesn't work as expected.

**Review by Aschepikov — APPROVED** [review]
GJ!

File: mailpoet/assets/js/src/email-editor/engine/editor.tsx [hunk 1]
```diff
@@ -33,6 +34,8 @@ function Editor() {
   );
 }
 
+const EditorWithPool = withNpsPoll(Editor);
```
Comment (oz) [inline mailpoet/assets/js/src/email-editor/engine/editor.tsx:37]: Did you mean EditorWith**Poll** instead of EditorWith**Pool**? 🤔

**Pull request merged**